heclib/heclib_c/src/System/getCurrentTimeMillis.c

```diff
@@ -32,6 +33,14 @@ long long getCurrentTimeMillis()
 {
 	long long ltime;
 
+	char* millis = getenv("DSS_CURRENT_TIME_MILLIS");
```

Comment:
Unless you're going to change the environment in the middle of the run I would cache the actual time as a static, with a bool to indicate it was provided. I don't think getenv is a cheap call and that's probably going to be a lot of atoi calls too.

Reply:
Granted, since performance, should probably be measured.

Reply:
good ideas -  I've updated the code.